data/json/items/tool_armor.json

```diff
@@ -178,6 +178,25 @@
     "use_action": "PORTABLE_GAME",
     "magazines": [ [ "battery", [ "light_minus_battery_cell", "light_minus_atomic_battery_cell", "light_minus_disposable_cell" ] ] ]
   },
+  {
+    "type": "TOOL_ARMOR",
+    "id": "fitness_band",
+    "name": "fitness band",
+    "name_plural": "fitness bands",
+    "category": "clothing",
+    "volume": "100 ml",
+    "description": "A fitness band that can track your heartbeat, exercise level and also has a clocks.  Activate to see your metrics.",
```

Comment:
Incorrect grammar here; correct one would be either "has a clock" or "has clocks"; likely the former.